tests/functional/test_uninstall.py

```diff
@@ -199,9 +199,11 @@ def test_uninstall_overlapping_package(script, data):
     assert_all_changes(result2, result3, [])
 
 
-@pytest.mark.parametrize("console_scripts",
-                         ["test_ = distutils_install",
-                          "test_:test_ = distutils_install"])
+@pytest.mark.parametrize("console_scripts", [
+    "test_ = distutils_install",
+    "test_:test_ = distutils_install",
+    "Test = distutils_install",
+])
 def test_uninstall_entry_point(script, console_scripts):
     """
     Test uninstall package with two or more entry points in the same section,
```

Comment:
What does this description actually mean? It seems like this is testing something very specific related to the hard-coded data below.

Reply:
Huh, didn’t notice that, thanks. I guess I’ll also change the function name to make the intention of this test clearer.